src/statement.ts

```diff
@@ -75,6 +76,21 @@ export class VariableStmt<
   }
 }
 
+export type VariableListParent = ForStmt;
+
+export class VariableList extends BaseNode<"VariableList", VariableListParent> {
+  readonly nodeKind: "Node" = "Node";
+
+  constructor(readonly decls: VariableStmt[]) {
```

Comment:
It shouldn't be `VariableStmt`inside the list.

It's

```ts
VariableStmt(VariableDeclList(VariableDecl[]))
```

Reply:
yeah, `VariableStatement` and `VariableDeclaration` are currently merged. `VariableStmt` matches the `VariableDeclaration`.

In a for loop, it contains a `VariableDeclarationList` -> `VariableDeclaration`.